5.6/alpine/Dockerfile

```diff
@@ -0,0 +1,70 @@
+FROM alpine:3.3
+
+ENV PHP_INI_DIR /usr/local/etc/php
+RUN mkdir -p $PHP_INI_DIR/conf.d
+
+ENV PHP_VERSION 5.6.19
+ENV PHP_FILENAME php-5.6.19.tar.xz
+ENV PHP_SHA256 2a24a3f84971680ac0a4c71050067de4f76ee235aa4a041fae21bfa69975c168
+
+##<autogenerated>##
+##</autogenerated>##
+
+# --enable-mysqlnd is included below because it's harder to compile after the fact the extensions are (since it's a plugin for several extensions, not an extension in itself)
+ENV GPG_KEYS 0BD78B5F97500D450838F95DFE857D9A90D90EC1 6E4F6AB321FDC07F2C332E3AC2BF0BC433CFC8B3
+RUN set -xe \
+	&& apk add --no-cache --virtual .build-deps \
+		autoconf \
+		curl-dev \
+		file \
+		gcc \
+		gnupg \
+		libc-dev \
+		libedit-dev \
+		libxml2-dev \
+		make \
+		openssl-dev \
+		pkgconf \
+		re2c \
+		sqlite-dev \
+	&& for key in $GPG_KEYS; do \
+		gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "$key"; \
+	done \
+	&& curl -fSL "http://php.net/get/$PHP_FILENAME/from/this/mirror" -o "$PHP_FILENAME" \
+	&& echo "$PHP_SHA256 *$PHP_FILENAME" | sha256sum -c - \
+	&& curl -fSL "http://php.net/get/$PHP_FILENAME.asc/from/this/mirror" -o "$PHP_FILENAME.asc" \
+	&& gpg --batch --verify "$PHP_FILENAME.asc" "$PHP_FILENAME" \
+	&& mkdir -p /usr/src \
+	&& tar -Jxf "$PHP_FILENAME" -C /usr/src \
+	&& mv /usr/src/php-$PHP_VERSION /usr/src/php \
+	&& rm "$PHP_FILENAME"* \
+	&& cd /usr/src/php \
+	&& ./configure \
+		--with-config-file-path="$PHP_INI_DIR" \
+		--with-config-file-scan-dir="$PHP_INI_DIR/conf.d" \
+		$PHP_EXTRA_CONFIGURE_ARGS \
+		--disable-cgi \
+		--enable-mysqlnd \
+		--with-curl \
+		--with-openssl \
+		--with-libedit \
+		--with-zlib \
+	&& make -j"$(getconf _NPROCESSORS_ONLN)" \
+	&& make install \
+	&& { find /usr/local/bin /usr/local/sbin -type f -perm +0111 -exec strip --strip-all '{}' + || true; } \
+	&& make clean \
+	&& runDeps="$( \
+		scanelf --needed --nobanner --recursive /usr/local \
+			| awk '{ gsub(/,/, "\nso:", $2); print "so:" $2 }' \
+			| sort -u \
+			| xargs -r apk info --installed \
+			| sort -u \
+	)" \
+	&& apk add --virtual .php-rundeps $runDeps \
+	&& apk del .build-deps
+
+COPY docker-php-ext-* /usr/local/bin/
+
+##<autogenerated>##
+CMD ["php", "-a"]
```

Comment:
Nope, we want to keep official images consistent so that a `docker run -it --rm random-image valid-shell` works: https://github.com/docker-library/official-images#consistency.  If there is a need for an entrypoint script to do something else like a [`chown`](https://github.com/docker-library/elasticsearch/blob/master/docker-entrypoint.sh#L14), then it could be added in the way described in the official images.